src/draw/vg_lite/lv_draw_vg_lite_vector.c

```diff
@@ -371,30 +369,30 @@ static void lv_path_to_vg(lv_vg_lite_path_t * dest, const lv_vector_path_t * src
     for(uint32_t i = 0; i < op_size; i++) {
         switch(ops[i]) {
             case LV_VECTOR_PATH_OP_MOVE_TO: {
-                    VLC_SET_OP_CODE(path_data++, VLC_OP_MOVE);
+                    LV_VG_LITE_PATH_SET_OP_CODE(path_data++, uint32_t, VLC_OP_MOVE);
```

Comment:
This is something that I get confused sometimes, the path_data next point is getting incremented after each operation added to the path buffer, but there are no bound check after every operation, is there a guarantee that the next free point in the path buffer will be always pointing of a valid bound of the path buffer?

Reply:
No boundary check required. Because the previous `lv_vg_lite_path_reserve_space` ensures that enough space is allocated ahead of time for safe writing, reducing repeated boundary checks can significantly improve efficiency, especially if the path data is long.
In addition, I also call `LV_ASSERT_MSG` after the loop to check whether the converted path matches the calculated path exactly, and the program will report an error if it does not match.

Reply:
Perfect, thank you for the clarification @FASTSHIFT